gno.land/pkg/gnoclient/client_queries.go

```diff
@@ -123,3 +125,55 @@ func (c *Client) QEval(pkgPath string, expression string) (string, *ctypes.Resul
 
 	return string(qres.Response.Data), qres, nil
 }
+
+// Block gets the latest block at height, if any
+// Height must be larger than 0
+func (c *Client) Block(height int64) (*ctypes.ResultBlock, error) {
+	if err := c.validateRPCClient(); err != nil {
+		return nil, ErrMissingRPCClient
+	}
+
+	if height <= 0 {
+		return nil, ErrInvalidBlockHeight
+	}
```

Comment:
The condition that height > 0 is not enforced here

Reply:
Not sure I fully get what you're trying to say- do you want me to change the condition to `height < 0`?

Reply:
I misread the condition, it's good, it enforces that height is > 0 👍 

Resolving